doc/specs/#7335 - Console Allocation Policy.md

```diff
@@ -117,19 +158,24 @@ This should have no impact on reliability.
 
 ### Compatibility
 
-On downlevel versions of Windows that do not understand (or expect) this manifest field, applications will allocate
-consoles as specified by their image subsystem (described in the [abstract](#abstract) above).
-
-An existing application opting into **inheritOnly** may constitute a breaking change, but the scope of the breakage is
+An existing application opting into **hidden** may constitute a breaking change, but the scope of the breakage is
 restricted to that application and is expected to be managed by the application.
 
 All behavioral changes are opt-in.
 
-> **EXAMPLE**: If Python updates python.exe to specify an allocation policy of **inheritOnly**, graphical python
-> applications will become double-click runnable from the graphical shell without spawning a console window. _However_,
-> console-based python applications will no longer spawn a console window when double-clicked from the graphical shell.
+> **EXAMPLE**: If Python updates python.exe to specify an allocation policy of **hidden**, graphical python applications
+> will become double-click runnable from the graphical shell without spawning a console window. _However_, console-based
+> python applications will no longer spawn a console window when double-clicked from the graphical shell.
+>
+> If python.exe specifies **detached**, Console APIs will fail until a console is allocated.
 
-Python could work around this by calling [`AllocConsole`] if it can be detected that console I/O is required.
+Python could work around this by calling [`AllocConsole`] if it can be detected that console I/O is required, or
+`GetConsoleWindow`/`ShowWindow` if it can detect that the user wants to interact with the hidden console session.
+
+#### Downlevel
+
+On downlevel versions of Windows that do not understand (or expect) this manifest field, applications will allocate
+consoles as specified by their image subsystem (described in the [abstract](#abstract) above).

```

Comment:
Okay so this is the big thing then - if python gets rid of `pythonw` and does this, then running GUI apps with the newly-manifested `python.exe` downlevel will still result in a console window appearing. They'd need to maintain `pythonw` as a downlevel shim still, correct?

Reply:
Yup. Unless we can service stuff.